### PR TITLE
feat(benchmarks): add Kontent benchmark

### PR DESCRIPTION
**Pull request by Simply007**
## Description

This adds a benchmarking setup for [Kontent](https://kontent.ai)

For data generation, I have used:
* https://github.com/Simply007/kontent-data-generator

Currently, I have projects with 512 and 4096 articles, I will generate 8k and 32k and provide all required env variables via some private channel.

### CODE REVIEW

File: benchmarks/source-kontent/.env.template [hunk 1]
```diff
@@ -0,0 +1,3 @@
+KONTENT_PROJECT_ID=4d1eb9ae-c95e-00e1-da99-692832fe675c
```
Comment (duffn) [inline benchmarks/source-kontent/.env.template:1]: Can you please prefix all environment variables with `BENCHMARK`? ex. `BENCHMARK_KONTENT_PROJECT_ID`. This helps us identify and organization benchmark related env vars.

Comment (Simply007) [reply benchmarks/source-kontent/.env.template:1]: For sure!

Comment (Simply007) [reply benchmarks/source-kontent/.env.template]: bdefa5a

**Comment by Simply007** [comment]
Hey guys,

I have just tried build for 32k pages and it took ~36mins, which is pretty cool I think.

I don't know who should I share the project keys with: @duffn, or @DSchau?

**Comment by Simply007** [comment]
One more question about the update, the npm script `data-update` basically just append `!` to the last article title, but if you make this change frequently, you might hit the maximum allowed length - which should be 10k characters.

**Comment by duffn** [comment]
> One more question about the update, the npm script `data-update` basically just append `!` to the last article title, but if you make this change frequently, you might hit the maximum allowed length - which should be 10k characters.

Are we updating a random article every time? The benchmark will only be 1-2 times per day, so don't imagine this will become a problem.

**Comment by duffn** [comment]
> I don't know who should I share the project keys with: @duffn, or @DSchau?

Either of us is fine.

**Comment by Simply007** [comment]
> The benchmark will only be 1-2 times per day, so don't imagine this will become a problem.

Currently, it is the first article by article number: https://github.com/gatsbyjs/gatsby/pull/24605/files#diff-2d9a4ac01afa73be31e8d24532778f04R21

Do you want to have it random? 
I have implemented is based on drupal, which is getting the first created article:
https://github.com/gatsby-inc/drupal-updater/blob/master/src/updater.ts#L45

But I can definitely to update it to random (as a Sanity does):
https://github.com/gatsbyjs/gatsby/pull/24517/files#diff-c11783c10707e9768f19927f7a98031aR12

**Comment by duffn** [comment]
> But I can definitely to update it to random (as a Sanity does):
> [#24517 (files)](https://github.com/gatsbyjs/gatsby/pull/24517/files#diff-c11783c10707e9768f19927f7a98031aR12)

Yes, I think it would be good to make it a random article. Thanks!

**Comment by Simply007** [comment]
> Yes, I think it would be good to make it a random article. Thanks!

Done @duffn

**Comment by Simply007** [comment]
> @Simply007 really like the API for updating stuff here 👍 Kudos!
> 
> Think this looks good to go, but I'll leave final approval to @duffn

Thanks, @DSchau! 
If you are referring to JS SDK API we should definitely thanks @Enngage.

Looking forward for final approval.

**Pull request merged**